README.md

```diff
@@ -92,7 +92,8 @@ where `nextflow.config` include the minimum set of parameters to run ERVmap with
 params {
     genome='/path/to/genome'               # external path to the indexed genome for the STAR aligner
     inputDir='path/to/input/folder'        # external path of the input data
-    inputPattern="*{1,2}.fastq.gz"         # pattern to search for input FASTQ files
+    inputPattern="*{1,2}.fastq.gz"         # pattern to search for input FASTQ files, or BAM files (*.{bam,bam.bai})
```

Comment:
Since they are constants, especially for patterns, use single quote (to avoid string expansion).

Reply:
Ok, I though I needed the expansion.